Elasticsearch/Indexer.php

```diff
@@ -95,4 +95,9 @@ private function getClient(): Client
     {
         return $this->factory->fromConfig($this->options);
     }
+
+    public function getAliases($params = array()): array
```

Comment:
Do we want to pass an array with Params or only the index name as a string? Maybe we should convert this to “getAliasByIndexName” to make the call easier?

Reply:
OK good idea